### ISSUE TITLE
Make release v1.0

### ISSUE THREAD

**Issue by DominicOram** [DiamondLightSource/python-artemis#493]
We need a release for the multi-crystal testing on 26/01/23.

Tickets that need to be merged first:
- [x] Some version of https://github.com/DiamondLightSource/python-artemis/issues/451
- [x] https://github.com/DiamondLightSource/python-artemis/issues/438
- [x] https://github.com/DiamondLightSource/python-artemis/pull/483 - highlight this is a breaking change in release notes

Nice to haves:
- [ ] https://github.com/DiamondLightSource/python-artemis/pull/466
- [ ] https://github.com/DiamondLightSource/python-artemis/pull/467
- [ ] https://github.com/DiamondLightSource/python-artemis/pull/495

**Comment by DominicOram** [comment]
On beamline and tested, only issue was https://github.com/DiamondLightSource/python-artemis/issues/436
